kafka_consumer_freshness_tracker/src/test/java/com/tesla/data/consumer/freshness/BurrowTest.java

```diff
@@ -121,6 +121,24 @@ public void testFailGroupLookupWhenResponseIsNotAMap() throws IOException {
     burrow.getConsumerGroupStatus("mycluster", "mygroup");
   }
 
+  @Test
+  public void testGetClusterBootstrapServers() throws Exception {
+    OkHttpClient client = Mockito.mock(OkHttpClient.class);
+    List<String> expected = Lists.newArrayList("kafka01.example.com:10251", "kafka02.example.com:10251");
+    Map<String, List<String>> module = new HashMap<>();
+    module.put("servers", expected);
+    Map<String, Object> cluster = new HashMap<>();
+    cluster.put("module",module);
+    Map<String, Object> response = new HashMap<>();
+    response.put("cluster", cluster);
+    when(client.newCall(any())).then(respondWithJson(response));
+
+    Burrow burrow = new Burrow(CONF, client);
+    List<String> servers = burrow.getClusterBootstrapServers("cluster_name");
+    assertEquals(2, servers.size());
+    assertEquals(expected, servers);
```

Comment:
you can just use this assert, you don't need the one above. In fact, the one above makes it harder to debug, as it will just show the sizes don't match, but not what the actual difference is.